pkg/util/virtualmachines/liststopped_test.go

```diff
@@ -0,0 +1,240 @@
+package virtualmachines
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"errors"
+	"reflect"
+	"testing"
+
+	mgmtcompute "github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2019-03-01/compute"
+	"github.com/Azure/go-autorest/autorest/to"
+	"github.com/golang/mock/gomock"
+
+	mock_compute "github.com/Azure/ARO-RP/pkg/util/mocks/azureclient/mgmt/compute"
+)
+
+func TestListStopped(t *testing.T) {
```

Comment:
This test covers almost everything that `pkg/install/start_vms_test.go` does. It's just missing expected calls to `StartAndWait`. The original test in `install` could now probably be simplified or maybe even deleted.